bin/npg_publish_tree.pl

```diff
@@ -168,7 +167,8 @@ sub _read_metadata_file {
 }
 
 if (defined $metadata_file and $stdio) {
-  $log->logconfess('Metadata json file and metadata from stdin options cannot be specified together');
+  $log->logconfess('Metadata JSON file and metadata from STDIN options' .
```

Comment:
You need to leave a space at the end of the first line if splitting this way. When joined, they concatentate works in the error message. A shown in the original `logcroak`, you don't need to explicitly concatentate with `.`, instead you can pass multiple strings to the logging methods.